src/app/containers/ConsentBanner/index.canonical.jsx

```diff
@@ -1,9 +1,10 @@
-import React, { useState, useEffect, useContext } from 'react';
+import React, { useState, useEffect, useContext, useRef } from 'react';
+import { oneOfType, func, shape, any } from 'prop-types';
 import Banner from './Banner/index.canonical';
 import consentBannerUtilities from './CanonicalLogic';
 import { UserContext } from '#contexts/UserContext';
 
-const Canonical = () => {
+const Canonical = ({ brandRef }) => {
```

Comment:
```suggestion
const Canonical = ({ onDismissFocusRef }) => {
```
Another naming thing, just decoupling the consent banner from the brand component, it could focus on any component on dismiss in the future if requirements change.